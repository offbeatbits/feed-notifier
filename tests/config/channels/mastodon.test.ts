describe('Mastodon config', () => {
  it('should export Mastodon config', () => {
    expect(require('../../../src/config/channels/mastodon')).toMatchSnapshot(); // eslint-disable-line
  });
});
