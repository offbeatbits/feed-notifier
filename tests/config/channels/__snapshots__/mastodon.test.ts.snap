// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mastodon config should export Mastodon config 1`] = `
Object {
  "mastodon": Object {
    "enabled": true,
    "host": "https://mastodon.test.example.org",
    "username": "test_account",
  },
}
`;
